addToCollection.php

```diff
@@ -0,0 +1,104 @@
+<?php
+require_once 'functions.php';
+
+$db = getDB();
+$cleansedArray = cleanseData($_POST);
+
+if(!empty($cleansedArray)) {
+    if (checkItemExists($cleansedArray, $db) != []) {
+        echo '<p>This item already exists in the collection.</p>';
```

Comment:
Assign this to a variable that gets echo'd out somewhere a little more visible - perhaps under the form. Can use the null coaelescer to decide wether there is something to display or not. (take a look at Milo's example of this if not sure)